course_content/notebooks/dask_intro.ipynb

```diff
@@ -831,6 +831,94 @@
    "cell_type": "raw",
    "metadata": {},
    "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using `filter` as a Constraint for Iris Load with Dask\n",
+    "\n",
+    "We have seen how to load a subset of data by passing a `constraint` to `iris.load` when loading from `files`. We have also seen how to do this by passing lists of filenames and constraints to `delayed(iris.load)`, generating a `dask.bag`. Here we will use the `map` and `filter` functionality of `dask.bag` with `iris.load` to subset data using a phenomenon constraint."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we should define what we mean when we talk about `map` and `filter`:\n",
+    "\n",
+    "`map(func, seq)`: A function which takes a function (`func`) and a sequence of inputs (`seq`) as arguments. It returns a sequence of outputs which are the result of applying `func` to each of the items in `seq`.\n",
+    "\n",
+    "`filter(func, seq)`: A function which also takes a function (`func`) and a sequence of inputs (`seq`) as arguments. `func` must be a function which returns a Boolean value, i.e. either `True` or `False`. This function will be applied to every element of `seq` and only if `func` returns `True` will the element of the sequence be included in the output sequence."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`filter` sounds very similar to what we were doing when using `iris.load` with a `constraint`. In a similar way to how we generated `bag_o_cubes` we can generate a `dask.bag` by mapping a sequence of filenames onto `iris.load`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bag_of_cubes = db.from_sequence(files).map(iris.load)\n",
+    "bag_of_cubes.compute()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can use the `filter` function of a `bag_of_cubes` to constrain it based on the phenomenon it represents. To do this we need to define a filter function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "phenom_filter = lambda cube: cube.name() == 'air_temperature'"
```

Comment:
We've used `air_potential_temperature` elsewhere, would you mind making this consistent?